### DISCUSSION TITLE
2022-11-02-chokuretsu-archives

### DISCUSSION THREAD

**Discussion by giscus** [haroohie-club/HaroohieSite#39] [Announcements]
# 2022-11-02-chokuretsu-archives

对 Haroohie Translation Club 博客的简要介绍，并解释我们翻译凉宫春日游戏的动机。

https://haroohie.club/blog/2022-10-17-welcome-why

**Comment by gboooo** [comment]
sub_201D310 is the sdk's u32 by u32 division builtin
